main.tf

```diff
@@ -118,3 +118,29 @@ module "ansible" {
   envs      = ["host=${aws_eip.default.public_ip}"]
   playbook  = "${var.ansible_playbook}"
 }
+
+# Restart dead or hung instance
+
+resource "null_resource" "check_alarm" {
+  triggers = {
+    action = "arn:aws:swf:${var.aws_region}:${var.aws_account_id}:${var.default_alarm_action}"
+  }
+}
+
+resource "aws_cloudwatch_metric_alarm" "reboot" {
+  alarm_name          = "${module.label.id}-status-check-failed"
+  comparison_operator = "${var.comparison_operator}"
```

Comment:
Shouldn't we hardcode these variables?
@osterman what do you think?

Reply:
I probably would have just hard coded it, but since @SweetOps has already parameterized it, I'm okay with it so long as we rename `reboot` to `default` or something more general.